### PR TITLE
Add tests for the exception and not found views.

### PR DESCRIPTION
**Pull request by bowlofeggs**
Signed-off-by: Randy Barlow <randy@electronsweatshop.com>

### CODE REVIEW

**Comment by bowlofeggs** [comment]
jenkies test

**Comment by bowlofeggs** [comment]
jenkies test

**Comment by bowlofeggs** [comment]
jenkies test

**Comment by bowlofeggs** [comment]
jenkies test

File: bodhi/tests/server/functional/test_generic.py [hunk 1]
```diff
@@ -697,3 +711,13 @@ def test_home_counts(self):
         </a>
           <span class="label label-default">prerelease</span>
       </h3>""", res)
+
+
+class TestNotfoundView(base.BaseTestCase):
+    """Test the notfound_view() handler."""
+    def test_notfound(self):
+        """Assert that we correctly deal with 404's."""
+        res = self.app.get('/makemerich', status=404)
```
Comment (pypingou) [inline bodhi/tests/server/functional/test_generic.py]: Well tried :) but you won't fool Bodhi that easily!

**Comment by pypingou** [comment]
👍 for me

File: bodhi/tests/server/functional/test_generic.py (hunk 1 above)
Comment (bowlofeggs) [reply bodhi/tests/server/functional/test_generic.py]: @pypingou hehe

**Pull request merged**